src/raytracer/Raytracer.m

```diff
@@ -244,26 +272,147 @@
     clear outputPAA
 end
 
+if paraCfgInput.nodeMobility
+else
+    outputPaaTime(:,:,2:paraCfgInput.numberOfTimeDivisions) = repmat(outputPaaTime(:,:,1), [1 1 paraCfgInput.numberOfTimeDivisions-1]);
+    Mpc(:,:,:,:,:,3:end) = repmat(Mpc(:,:,:,:,:,2), [1 1 1 1 1 paraCfgInput.numberOfTimeDivisions-1]);
+end
+
+%% Node-target ray tracing
```

Comment:
it looks like this could be easily separated to a dedicated function, to declutter Raytracer.m which already has lots of things to take care of

Reply:
Well, this code might be a separate function but it wouldn't be coherent with the rest of the code. This is still raytracing, just changing the input of the raytracing. I would keep it as is. 
The only option I see, it would be to have 2 functions, e.g., targetUnrealeatedRT (clustering the conventional node-to-node RT) and the targetRelatedRT (clustering the node-to-target RT)